### PR TITLE
Add a script to download all releases

### PR DESCRIPTION
**Pull request by cpitclaudel**
This is a small script that downloads all Dafny releases from GitHub (having a
bunch of releases on hand is convenient for getting a rough estimate of when a
bug was introduced).

### CODE REVIEW

**Review by atomb — DISMISSED** [review]
This is very nice. I would have found this useful recently. I especially like that it automatically extracts the archives into versioned directories, which is one of the most labor-intensive aspects of doing this manually.

(In the longer term it makes me wonder whether we should do Dafny releases with a path prefix that includes the version number, as most open source projects do.)

File: Scripts/download_all_releases.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    except:
+        shutil.rmtree(target_dir)
+        raise
+
+def get_platform_glob() -> str:
+    system = platform.system()
+    if system == 'Linux':
+        return '*ubuntu*'
+    if system == 'Darwin':
+        return '*mac*'
```
Comment (wilcoxjay) [inline Scripts/download_all_releases.py:83]: this should be `*osx*` I think

Comment (cpitclaudel) [reply Scripts/download_all_releases.py:83]: Good catch, thanks!

Comment (cpitclaudel) [inline Scripts/download_all_releases.py:83]:
```suggestion
        return '*osx*'
```

**Review by atomb — APPROVED** [review]

**Pull request merged**